update.rb

```diff
@@ -127,6 +127,26 @@
 
   updated_files = updater.updated_dependency_files
 
+  g = Gitlab.client(
+    endpoint: source.api_endpoint,
+    private_token: ENV["KIRA_GITLAB_PERSONAL_TOKEN"]
+  )
+
+  # Is there an open MR for this dependency version, then close it
+  open_merge_requests = g.merge_requests(repo_name, state: "opened")
+  open_merge_requests.each do |omr|
+    title = omr.title
+    if title.include?(dep.name) && title.include?(dep.version) && !title.include?(updated_deps.first.version)
+
+      # close the merge request
+      g.update_merge_request(repo_name, omr.iid, {state_event: "close"})
+      puts "closed ##{omr.iid}"
+
+      # delete the branch
+      g.delete_branch(repo_name, omr.source_branch)
+    end
```

Comment:
Looks good! Have you tested it locally? (Sorry for the lack of unit tests, that's my first ruby project 😆 )

Reply:
Yep I've tested it on 2 of my repos and it works well. Here's a screenshot of it where it's closed an old MR :)

![Screenshot from 2019-07-31 10-08-10](https://user-images.githubusercontent.com/425200/62199341-369b2f80-b37b-11e9-8d9f-20500269f3c9.png)

Reply:
AWESOME 🎉